modules/dxp/apps/search-experiences/search-experiences-rest-impl/src/main/java/com/liferay/search/experiences/rest/internal/resource/v1_0/SXPElementResourceImpl.java

```diff
@@ -422,6 +390,23 @@ private String _getSchemaVersion() {
 		return "1.0";
 	}
 
+	private SXPElement _putSXPElement(
+			SXPElement sxpElement,
+			com.liferay.search.experiences.model.SXPElement sxpElementModel)
```

Comment:
Please rename sxpElementModel -> "serviceBuilderSXPElement"

Also Please check the casing for serviceBuilderSXPElement in the new put methods.